railties/lib/rails/generators/rails/controller/controller_generator.rb

```diff
@@ -3,13 +3,16 @@
 module Rails
   module Generators
     class ControllerGenerator < NamedBase # :nodoc:
-      argument :actions, type: :array, default: [], banner: "action action"
+      argument :args_actions, type: :array, default: [], banner: "action action"
```

Comment:
This changes the output of the help. I'd do the opposite. I think you can override the reader of `actions` without having to rename the argument. Is not that the case?

Reply:
There not change in for the help output related. Anyway, your point is right, I'll try override the reader to this arguments and this will more clean.